boards/doit/dt_bl10_devkit/board.yml

```diff
@@ -0,0 +1,4 @@
+board:
```

Comment:
Would be nice to have a vendor field defined in this yaml as well.

Reply:
Noted! I removed this board from PR to reduce the scope.